### ISSUE TITLE
Missing DLLs when launching application on Windows

### ISSUE THREAD

**Issue by CloudViking86** [pgmodeler/pgmodeler#1439]
TL;DR:
Please modify the code for "Resolving dependencies" for Windows on the "Build and install pgModeler"-page here since I believe it doesn't state all the required DLLs in the "Platform specific requirements (Windows")-section.

---

OS: Windows 10 Pro v. 1909 (Build 18363.836)
Version of "MSYS2": msys2-x86_64-20200522
Version of pgmodeler: 0.9.2
Path of extracted source: C:\msys64\home\CloudViking86\pgmodeler-0.9.2\
Path of intended compiled build: C:\pgmodeler\
Path of "msys64": C:\msys64\

Steps:

1) Downloaded "MSYS2"
2) Ran the following commands in the "MSYS2 MinGW 64-bit" terminal;
```
pacman -Suy
pacman -Suy
pacman -S base-devel mingw-w64-x86_64-make mingw-w64-x86_64-gcc mingw-w64-x86_64-postgresql mingw-w64-x86_64-qt5
```
3) Downloaded the .tar of pgmodeler 0.9.2 (pgmodeler-0.9.2.tar) and unzipped
4) Content of modified "pgmodeler.pri":
```
windows {
  !defined(PGSQL_LIB, var): PGSQL_LIB = C:/msys64/mingw64/bin/libpq.dll
  !defined(PGSQL_INC, var): PGSQL_INC = C:/msys64/mingw64/include
  !defined(XML_INC, var): XML_INC = C:/msys64/mingw64/include/libxml2
  !defined(XML_LIB, var): XML_LIB = C:/msys64/mingw64/bin/libxml2-2.dll
...
```
5) Running "qmake" in "MSYS2 MiniGW 64-bit" when standing in "/home/CloudViking86/pgmodeler-0.9.2" which is actually "C:\msys64\home\CloudViking86\pgmodeler-0.9.2":
`qmake -r CONFIG+=release PREFIX=C:/pgbuilder pgmodeler.pro > qmake.log 2>&1`
https://pastebin.com/10kjNHeN
6) Running "make" (same dir as above):
`make > make.log 2>&1`
https://pastebin.com/i0NYmRg5
7) Running "make install" (same dir as above):
`make install > make_install.log 2>&1`
https://pastebin.com/7KhHpgEs
8) switching dir to installation dir / dir where compiled app should be located and running "windeployqt":
`cd /c/pgbuilder`
`windeployqt pgmodeler.exe pgmodeler_ui.dll`
https://pastebin.com/r7FyVzSZ
9) switching dir to "mingw64/bin" in "msys64" and copying various DLLs:
`cd /mingw64/bin/`
`cp libicuin*.dll libicuuc*.dll libicudt*.dll libpcre2-16-0.dll libharfbuzz-0.dll \
      libpng16-16.dll libfreetype-6.dll libgraphite2.dll libglib-2.0-0.dll libpcre-1.dll \
      libbz2-1.dll libssl-1_1-x64.dll libcrypto-1_1-x64.dll libgcc_s_seh-1.dll \
      libstdc++-6.dll libwinpthread-1.dll zlib1.dll libpq.dll libxml2-2.dll liblzma-5.dll \
      libiconv-2.dll libintl-8.dll C:/pgbuilder`
10) Starting "pgmodeler.exe" in "C:\pgbuilder":

pgmodeler.exe - System Error
The code execution cannot proceed because libdouble-conversion.dll was not found. Reinstalling the program may fix this problem.
^
Copying that file from "C:\msys64\mingw64\bin" to "C:\pgbuilder\"

Same error and procedure for the following DLLs:
libzstd.dll
libbrotlidec.dll
libbrotlicommon.dll

After copying the missing DLLs the application launches successfully, haven't started using it so can't vouch that it works 100%.

Just wanted to bring that to your attention (and started saving all my logs before I got it working by simply copying the missing DLLs :P)

Best Regards / CloudViking86

**Comment by rkhaotix** [comment]
Thanks for that @CloudViking86 

Unfortunately, MSYS2 constantly includes extra dependencies to its Qt build and sometimes I can't update the installation procedures accordingly quicly. I'll update them ASAP. :)

**Comment by CloudViking86** [comment]
> Thanks for that @CloudViking86
> 
> Unfortunately, MSYS2 constantly includes extra dependencies to its Qt build and sometimes I can't update the installation procedures accordingly quicly. I'll update them ASAP. :)

No worries, I am just grateful for the instructions on how to compile it on Windows, very detailed and precise and not often you see that 👍 Totally understand. Keep up the good work!
